src/components/Registration/RegAddEditOrgPerson.vue

```diff
@@ -111,11 +111,11 @@
                   v-model="orgPerson.confirmBusiness"
                   :rules="enableRules ? [(v) => !!v] : []"
                 >
-                  <template v-if="isProprietor" slot="label">
+                  <template v-if="isProprietor" v-slot:label>
                     I confirm that the business proprietor being added is not legally required to
                     register in B.C.
                   </template>
-                  <template v-if="isPartner" slot="label">
+                  <template v-else-if="isPartner" v-slot:label>
```

Comment:
There was a warning/error that an element cannot have multiple label slots... This was because it was not clear that these conditions cannot both be true. The v-else-if fixes that.